CMakeLists.txt

```diff
@@ -33,13 +36,18 @@ set(FREESURROUND_HEADERS src/addon.h
                          src/kissfft/kiss_fft.h
                          src/kissfft/kiss_fftnd.h
                          src/kissfft/kiss_fftndr.h
-                         src/kissfft/kiss_fftr.h)
+                         src/kissfft/kiss_fftr.h
+                         )
 
 set(DEPLIBS ${kodiplatform_LIBRARIES}
             ${p8-platform_LIBRARIES})
 
+addon_source_group("${FREESURROUND_SOURCES}")
+addon_source_group("${FREESURROUND_HEADERS}")
```

Comment:
Hehe no problem.

I temporarily removed them because it is a limitation from our build system. Headers are currently not added to the Visual Studio solution. I guess the same will count for XCode.